models/viralspread/netlogo/LeafNetwork_Robert_mod.nlogo

```diff
@@ -1,3 +1,15 @@
+globals
+[
+  total-leaf1
+  total-leaf2
+  total-leaf3
+  total-leaf4
+  infected-leaf1
+  infected-leaf2
+  infected-leaf3
+  infected-leaf4
+]
+
```

Comment:
NetLogo can do arrays, yeah, though the primary data structure is lists. 
Arrays may be better when we have an aribitrary number leaves. But for now, just having a few global variables, updating their values and printing to file does what we want.